securesystemslib/metadata.py

```diff
@@ -94,3 +90,70 @@ def pae(self) -> bytes:
             len(self.payload),
             self.payload,
         )
+
+    def sign(self, signer: Signer) -> Signature:
+        """Sign the payload and create the signature.
+
+        Arguments:
+            signer: A "Signer" class instance.
+
+        Returns:
+            A "Signature" instance.
+        """
+
+        signature = signer.sign(self.pae())
+        self.signatures.append(signature)
+
+        return signature
+
+    def verify(self, keys: List[Key], threshold: int) -> Dict[str, Key]:
+        """Verify the payload with the provided Keys.
+
+        Arguments:
+            keys: A list of "Key" class instance that is a public key to verify
+                the signatures.
+            threshold: Number of signatures needed to pass the verification.
+
+        Raises:
+            ValueError: If "threshold" is not valid.
+            SignatureVerificationError: If the amount of "recognized_signers" is less
```

Comment:
```suggestion
            SignatureVerificationError: If the amount of "accepted_keys" is less
```

Reply:
Actually, `accepted_keys` is an implementation detail and does not need to be mentioned in the docstring. Maybe you can rephrase the sentence without using the variable name.